packages/pulp/python-six/python-six.spec

```diff
@@ -0,0 +1,47 @@
+# Created by pyp2rpm-3.3.3
+%global pypi_name six
+
+Name:           python-%{pypi_name}
+Version:        1.13.0
+Release:        1%{?dist}
+Summary:        Python 2 and 3 compatibility utilities
+
+License:        MIT
+URL:            https://github.com/benjaminp/six
+Source0:        https://files.pythonhosted.org/packages/source/s/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
+BuildArch:      noarch
+
+BuildRequires:  python3-devel
+BuildRequires:  python3-setuptools
+
+%description
+ Six is a Python 2 and 3 compatibility library. It provides utility functions
+
+%package -n     python3-%{pypi_name}
+Summary:        %{summary}
+%{?python_provide:%python_provide python3-%{pypi_name}}
+
+%description -n python3-%{pypi_name}
+ Six is a Python 2 and 3 compatibility library. It provides utility functions
+
+%prep
+%autosetup -n %{pypi_name}-%{version}
+# Remove bundled egg-info
+rm -rf %{pypi_name}.egg-info
+
+%build
+%py3_build
+
+%install
+%py3_install
+
+%files -n python3-%{pypi_name}
+%license LICENSE
+%doc README.rst
+%{python3_sitelib}/__pycache__/*
```

Comment:
Can't this include the `pypi_name` to be explicit?

Reply:
fixed by not building the c code :)

Reply:
that's what's in the default pyp2rpm template, I didn't investigate further.